includes/discovery/os/zxr10.inc.php

```diff
@@ -4,4 +4,7 @@
     if (preg_match('/^ZXR10/', $sysDescr)) {
         $os = 'zxr10';
     }
+    if (str_contains($sysDescr, "ZTE Ethernet Switch")) {
```

Comment:
any reason not to  use `str_contains($sysDescr, 'ZXR10')` and drop the preg_match above too?